src/work-allocation-2/components/constants/config.constants.ts

```diff
@@ -118,17 +118,22 @@ const ASSIGNEE: FieldConfig = {
 /**
  * The views.
  */
-const AVAILABLE_TASKS: FieldConfig[] = [
+const AVAILABLE_TASKS_FOR_JUDICIAL: FieldConfig[] = [
+  CASE_NAME_AS_TEXT, CASE_CATEGORY, LOCATION, DERIVED_ICON, TASK_NAME_AS_TEXT, DUE_DATE
+];
+const AVAILABLE_TASKS_FOR_LEGAL_OPS: FieldConfig[] = [
   CASE_NAME_AS_TEXT, CASE_CATEGORY, LOCATION, DERIVED_ICON, TASK_NAME_AS_TEXT, DUE_DATE
 ];
-
 const MY_CASES: FieldConfig[] = [
   CASE_NAME_AS_LINK, JURISDICTION, CASE_CATEGORY, CASE_ROLE, START_DATE, END_DATE
 ];
 const MY_TASKS: FieldConfig[] = [
   CASE_REFERENCE_AS_LINK, CASE_NAME_AS_TEXT, CASE_CATEGORY, LOCATION, DERIVED_ICON, TASK_NAME_AS_TEXT, DUE_DATE
 ];
-const MY_WORK_TASKS: FieldConfig[] = [
+const MY_WORK_TASKS_FOR_JUDICIAL: FieldConfig[] = [
+  CASE_NAME_AS_LINK, CASE_CATEGORY, LOCATION, DERIVED_ICON, TASK_NAME_AS_LINK, DUE_DATE
+];
+const MY_WORK_TASKS_FOR_LEGAL_OPS: FieldConfig[] = [
   CASE_NAME_AS_LINK, CASE_CATEGORY, LOCATION, DERIVED_ICON, TASK_NAME_AS_LINK, DUE_DATE
 ];
```

Comment:
looks like these two fields configs have same setting, why they are splited to different views?

Reply:
This is a preparation for the other JIRAs. Please see description